From: Jon Severinsson <jon@severinsson.net>
Date: Fri, 11 Jul 2014 14:37:36 +0200
Subject: build-sys: Do not distribute generated emergency.service

It is already in nodist_systemunit_DATA and if it is
shipped, it contains the hardcoded path to systemctl
which will cause it to fail to start when
rootprefix != prefix and rootbindir != bindir.

(cherry picked from commit 3864c28549d742427fdf33026e522e9f10a7e4ec)
---
 Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 9525e74..025461b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -413,7 +413,6 @@ dist_sysctl_DATA = \
 dist_systemunit_DATA = \
 	units/graphical.target \
 	units/multi-user.target \
-	units/emergency.service \
 	units/emergency.target \
 	units/sysinit.target \
 	units/basic.target \
